packages/app/src/core/application.module.ts

```diff
@@ -5,47 +5,68 @@
 
 import { ipcMain } from "electron-better-ipc"
 import MuditaDeviceManager from "@mudita/pure"
+import { EventEmitter } from "events"
 import { DeviceService } from "App/backend/device-service"
 // TODO change module name to `KeyStorage`
 import { MetadataStore } from "App/metadata/services"
 import logger from "App/main/utils/logger"
 import { flags, Feature } from "App/feature-flags"
 import PureLogger from "App/main/utils/pure-logger"
-import { IndexStorageService } from "App/index-storage/services"
+import { IndexFactory } from "App/index-storage/factories"
 import {
   DataIndexInitializer,
   ControllerInitializer,
+  ObserverInitializer,
 } from "App/core/initializers"
 import { Module } from "App/core/types"
+import { FileSystemService } from "App/file-system/services/file-system.service.refactored"
+
+import { IndexStorageModule } from "App/index-storage/index-storage.module"
+import { DataSyncModule } from "App/data-sync/data-sync.module"
+import { ContactModule } from "App/contacts/contact.module"
+import { MessageModule } from "App/messages/message.module"
 
 export class ApplicationModule {
-  static modules: Module[] = []
+  public modules: Module[] = [
```

Comment:
Why no `BaseModule`? I see that modules extend via `BaseModule` where just `Module` isn't coupled with any.